internal/policy/verify.go

```diff
@@ -682,6 +682,14 @@ func verifyTagEntry(ctx context.Context, repo *git.Repository, policy *State, en
 		return err
 	}
 
+	if entry.TargetID != tagObj.Hash && entry.TargetID != tagObj.Target {
+		return fmt.Errorf("verifying RSL entry failed, %s", ErrUnauthorizedSignature.Error())
+	}
+
+	if entry.RefName != plumbing.NewTagReferenceName(tagObj.Name).String() {
+		return fmt.Errorf(plumbing.NewTagReferenceName(entry.RefName).String())
```

Comment:
error needs to be updated? Perhaps that it doesn't match. Also, you can just use entry.RefName afaict.

Reply:
Fixed